doc/cookbook/index.md

```diff
@@ -0,0 +1,145 @@
+# The SeqAn Cookbook {#cookbook}
+
+[TOC]
+
+This document provides example recipes on how to carry out particular tasks using the SeqAn functionalities in C++.
+It will take some time, but we hope to expand this document into containing numerous great examples. If you have suggestions for how to improve the Cookbook and/or examples you would like included, please feel free to contact us.
+
+#  Read sequence files
+\snippet doc/tutorial/introduction/introduction_file_input.cpp fileinput
+
+# Write a custom validator
```

Comment:
I don't understand how the custom validator is connected to the following descriptions, or is it completely independent of it?

Reply:
You are right, the recipes are independent of each other. I added a remark on the fact that they are not ordered.